core/templates/dev/head/galleries/gallery.html

```diff
@@ -166,7 +166,7 @@
                 Oops, it looks like your search didn't match any explorations.<br><br>
                 <!-- NOTE: If you change the links here, please make the corresponding change for the 'Create New Exploration' button. -->
                 {% if not has_fully_registered %}
-                  Would you like to <a href="{{gallery_login_redirect_url}}">create one</a>?
+                  Would you like to <a href="{{gallery_login_redirect_url}}" ng-click="onRedirectToLogin()">create one</a>?
```

Comment:
This looks like it will call registerStartLoginEvent('createExplorationButton') which seems wrong.